cmd/activator/main.go

```diff
@@ -68,6 +68,10 @@ type retryRoundTripper struct {
 	start    time.Time
 }
 
+func (rrt *retryRoundTripper) CalculateDelay(retries int, minRetryInterval time.Duration) time.Duration {
+	return time.Duration(int(minRetryInterval/time.Millisecond)*retries*retries) * time.Millisecond
```

Comment:
I believe this is quadratic, not exponential.  What we want is an aggressive retry during normal activation times, but a quickly growing retry interval thereafter.  Which is easier to achieve with exponential because of that hockey stick shape.

In my experience a small base like 1.3 is a good start.  With the retry index as the exponent.  Then multiply by the min retry.

E.g. `return time.Duration(int(minRetryInterval/time.Millisecond)*(1.3^retries)) * time.Millisecond`

It would look something like this.  (The actual numbers should be tuned, but the point is to keep the curve low and fast until we leave normal operating conditions.)

![image](https://user-images.githubusercontent.com/1103629/43968852-2b5a5440-9c7d-11e8-8862-ac99cb24b79f.png)

Reply:
Went for Base=1.3, MinRetry=100ms for now, giving me a progression as shown in the table:

![image](https://user-images.githubusercontent.com/1289147/43970662-96948dec-9cce-11e8-846c-49ee7b6bf1e2.png)